lib/nodelist.js

```diff
@@ -1,29 +1,28 @@
 define(['sorttable', 'snabbdom', 'helper'], function (SortTable, V, helper) {
   'use strict';
   V = V.default;
-
-  function getUptime(now, d) {
-    if (d.is_online && 'uptime' in d) {
-      return Math.round(d.uptime);
-    } else if (!d.is_online && 'lastseen' in d) {
-      return Math.round(-(now.unix() - d.lastseen.unix()));
-    }
-    return 0;
-  }
+  var now = new Date().getTime();
 
   function showUptime(uptime) {
-    var s = '';
-    uptime /= 3600;
+    var s = (now - uptime) / (60 * 1000);
```

Comment:
if uptime is undefined it will throw an error and you test for undefined in line 9